src/DurableTask.Netherite/Connections/ConnectionNameToConnectionStringResolver.cs

```diff
@@ -0,0 +1,76 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace DurableTask.Netherite
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Data;
+    using System.Text;
+    using Microsoft.Azure.Storage;
+    using Microsoft.Extensions.Logging.Abstractions;
+
+    /// <summary>
+    /// Resolves connection names by using a mapping from connection names to connection strings.
+    /// </summary>
+    public class ConnectionNameToConnectionStringResolver : ConnectionResolver
```

Comment:
In addition to my other suggestion around maintaining IoC pattern with the factory, maybe this type could be made `internal`. I like to keep public API surface to only what is necessary. Once something is public, it is hard to take it back.

Reply:
yes.